services/management/dbaas/kubernetes_server.go

```diff
@@ -357,6 +357,16 @@ func (k kubernetesServer) UnregisterKubernetesCluster(ctx context.Context, req *
 			return models.RemoveKubernetesCluster(t.Querier, req.KubernetesClusterName)
 		}
 
+		_, err = k.dbaasClient.StopMonitoring(ctx, &dbaascontrollerv1beta1.StopMonitoringRequest{
+			KubeAuth: &dbaascontrollerv1beta1.KubeAuth{
+				Kubeconfig: kubernetesCluster.KubeConfig,
+			},
+		})
+
+		if err != nil {
+			return errors.Wrap(err, "cannot stop monitoring")
```

Comment:
Let's for now just log the message and go on.

Reply:
Changed to log.Warn